### DISCUSSION TITLE
I see KafkaUser created but no secret

### DISCUSSION THREAD

**Discussion by purbanow** [strimzi/strimzi-kafka-operator#5719] [Q&A]
Hi, as in the title when I'm creating a KafkaUser and calling kubectl get KafkaUser -n  namespace I see KafkaUser but when I'm calling kubectl get secrets -n  namespace I don't see the corresponding secret.

**Comment by scholzj** [comment]
I think you will need to share the YAML and the logs if you want any help. Some of the things you can check?
* The `.status` section of the KafkaUser resource
* The `strimzi.io/cluster` label on the KafkaUser resource
